src/utils/ConfigHelper.ts

```diff
@@ -102,7 +98,6 @@ export const configHelperNetworks: ConfigHelperConfig[] = [
     networkId: 2021000,
     network: 'gaiaxtestnet',
     nodeUri: 'https://gaia-x.rpc',
-    metadataCacheUri: 'https://aquarius.gaiaxtestnet.oceanprotocol.com',
```

Comment:
not sure if gx testnet will be in main aqua as well, @alexcos20 can you confirm this?

Reply:
not for the moment..
probably we will run a separate aquarius for this

Reply:
so we can leave as is and deal with it once we know